### PR TITLE
KOGITO-5712 - Remove integration tests from productized profile

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Comment by cristianonicolai** [comment]
As discussed with @jstastny-cz @mareknovotny IT modules should not be included in the final offline maven repository,

**Review by jstastny-cz — APPROVED** [review]
LGTM.

**Review by mareknovotny — APPROVED** [review]

**Review by jstastny-cz — COMMENTED** [review]
Please consider handling IT modules exclusion using a new property, as in cristianonicolai/kogito-runtimes#3 .
This will enable to easily scope test execution to the `productized` scope only, keeping IT modules part of maven reactor. Goal is to run the build using both `-Dproductized -DexcludeITmodules` properties (not building IT modules) and then tests will use `productized` property only to deduce the maven reactor for IT tests execution (including IT modules).

**Review by ricardozanini — APPROVED** [review]

File: springboot/starters/kogito-serverless-workflow-spring-boot-starter/pom.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <!-- Out of the box CE support -->
     <dependency>
       <groupId>org.kie.kogito</groupId>
-      <artifactId>kogito-addons-springboot-cloudevents</artifactId>
+      <artifactId>kogito-addons-springboot-messaging</artifactId>
```
Comment (cristianonicolai) [inline springboot/starters/kogito-serverless-workflow-spring-boot-starter/pom.xml:24]: @ricardozanini fyi, looks fine?

**Comment by cristianonicolai** [comment]
@mareknovotny @ricardozanini @jstastny-cz please take another look as I refactor it to have two profiles as suggested by @jstastny-cz . Combining `productized` and  `excludeITModules` profiles are the artefacts that will end up in the offline Maven repo.
If running just `productized`, it will now also include the IT modules related to the artefacts that need to be productized.

File: springboot/starters/kogito-serverless-workflow-spring-boot-starter/pom.xml (hunk 1 above)
Comment (ricardozanini) [reply springboot/starters/kogito-serverless-workflow-spring-boot-starter/pom.xml:24]: sure!

**Review by ricardozanini — APPROVED** [review]

**Comment by cristianonicolai** [comment]
Jenkins rerun examples tests

**Pull request merged**